tests/app/views/test_briefs.py

```diff
@@ -521,6 +521,18 @@ def test_list_briefs_by_framework(self):
         assert res.status_code == 200
         assert len(data['briefs']) == 5, data['briefs']
 
+    def test_list_briefs_by_multiple_frameworks(self):
+        self.setup_dummy_briefs(1, status='draft')
+        self.setup_dummy_briefs(3, status='live', brief_start=2)
+
+        # we, um, don't have multiple brief-capable frameworks in the test dataset yet so this test is a bit of a stub
```

Comment:
I agree having a comment makes sense here, but it could be less informal in style.  I was going to suggest it would be nice to setup multiple brief-capable frameworks in the tests, but on reflection it's a bit of a headache for little gain at this point.